.github/workflows/slsa-builder-go.yml

```diff
@@ -357,24 +357,16 @@ jobs:
           echo "provenance generator is $PROVENANCE_BINARY"
 
           # Create provenance
-          DIGEST="$UNTRUSTED_BINARY_HASH" ./"$PROVENANCE_BINARY" > "$UNTRUSTED_BINARY_NAME.intoto"
+          DIGEST="$UNTRUSTED_BINARY_HASH" ./"$PROVENANCE_BINARY"
 
           # Sign provenance
-          UNSIGNED_PROVENANCE="$UNTRUSTED_BINARY_NAME.intoto" ./"$PROVENANCE_BINARY" > "$UNTRUSTED_BINARY_NAME.signed.intoto"
-          echo "::set-output name=signed-provenance-name::$UNTRUSTED_BINARY_NAME.signed.intoto"
-
-      - name: Upload the provenance
-        uses: actions/upload-artifact@82c141cc518b40d92cc801eee768e7aafc9c2fa2 # v2.3.1
-        with:
-          name: "${{ needs.build-dry.outputs.go-binary-name }}.intoto"
-          path: "${{ needs.build-dry.outputs.go-binary-name }}.intoto"
-          if-no-files-found: error
-          retention-days: 5
+          UNSIGNED_PROVENANCE="$UNTRUSTED_BINARY_NAME.intoto" ./"$PROVENANCE_BINARY"
```

Comment:
Let's add a comment that provenance `set-output` is done by the binary for two variables, provenance name and sign provenance name